package.json

```diff
@@ -181,7 +181,7 @@
             "Use preferred quote style set by `xml.preferences.quoteStyle` when formatting.",
             "Ignore quote style formatting."
           ],
-          "default": "preferred",
+          "default": "ignore",
           "description": "Set quote style rule when formatting.",
```

Comment:
If the new description from the README is okay, we could probably use the same description here too

Reply:
+1